tests/Feature/Api/CityControllerTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api;
+
+use App\Models\City;
+use App\Models\Country;
+use App\Models\User;
+use Laravel\Sanctum\Sanctum;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\TestCase;
+
+class CityControllerTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $adminUser = User::factory()->create();
+        Sanctum::actingAs($adminUser, ["HasAdminRole"]);
+    }
+
+    public function testCitiesArrayIsReturned(): void
+    {
+        City::factory()->count(3)->create();
+
+        $response = $this->getJson("/api/admin/cities");
+
+        $response->assertStatus(200)->assertJsonStructure([
```

Comment:
This is one place, where `assertStatus(200)` is used - in another places is used `assertOk()`. Please unify that